addons/account/static/src/xml/account_reconciliation.xml

```diff
@@ -144,7 +144,8 @@
 
 <div t-name="reconciliation.line.match">
     <div class="match_controls">
-        <input class="filter" placeholder="Filter..." type="text" value=""/>
+        <input class="filter o_input" placeholder="Filter..." type="text" value=""/>
+        <input class="btn btn-secondary btn-sm" type="button" value="Filter"/>
```

Comment:
why not use a button instead of an input?
I ask because I notice that we have other events that are bind on input. I haven't tested it but does it not pose problem with the following event? 
'blur input': '_onInputKeyup',

Reply:
good point, I'll change it if we are to keep the button but I'm not sure yet